File: MODUL-6/c/PRAK604-2310817210007-RAYMONDHARIYONO.c
#include <stdio.h>
#include <string.h>

void main(){
    char baris1[30],baris2[30];

    scanf(" %[^\n]", &baris1);
    scanf(" %[^\n]", &baris2);

    printf("\n");

    size_t SIZE_baris1 = strlen(baris1);
    size_t SIZE_baris2 = strlen(baris2);

    if(SIZE_baris1 == SIZE_baris2){

        size_t i = 0;
        size_t j = 0;
 
        int asterisk = 0;
        int hash = 0;

        while(i < SIZE_baris1 && j < SIZE_baris2){

            if(baris1[i] == ' ' && baris2[j] ==' '){
                i++;
                j++;
                continue;
            }
            if(baris1[i] == baris2[j]){
                printf("*");
                asterisk++;
            }
            else{
                printf("#");
                hash++;
            }
            i++;
            j++;
        }
        printf("\n* = %d\n",asterisk);
        printf("# = %d\n",hash);

        if (asterisk >= hash){
            printf("Pesan Asli");
        }
        else{
            printf("Pesan Palsu");
        }
    
}else{
        printf("Panjang kalimat berbeda, pesan palsu");
    }
}

File: MODUL-6/python/PRAK604-2310817210007-RAYMONDHARIYONO.py
i=0;j=0
pesanAsli = input();pesanPalsu = input()
kode = len(pesanAsli);pesan = len(pesanPalsu)
if kode != pesan :
    print('panjang Kalimat Berbeda, Pesan Palsu')
else :
    for x in range(kode):
        if(pesanAsli[x]==pesanPalsu[x]):
            if (pesanAsli[x] == ' '):
                print(" ", end = " ")
            else:
                print("*", end = " ")
                i += 1
        else :
            print("#",end=" ")
            j += 1
    print("\n* =",i)
    print("# =",j)
    if(i > j):
        print("Pesan Asli")
    else:
        print("Pesan Palsu")